README.md

```diff
@@ -101,6 +105,20 @@ to get official guidelines):
 # docker/images/foo/Dockerfile
 FROM foo:2.4.12
 
+# Build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.name="openshift-foo" \
+      org.label-schema.description="OpenShift-compatible foo Docker image" \
+      org.label-schema.url="https://github.com/openfun" \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.vcs-url="https://github.com/openfun/openshift-docker" \
+      org.label-schema.vendor="GIP FUN MOOC" \
+      org.label-schema.version=$VERSION \
+      org.label-schema.schema-version="1.0"
+
```

Comment:
I can add `ARG` statements with default values to the `Dockerfile` and hard-code them in the `bin/build` script to avoid repetition in our Dockerfiles. Will that be ok?

Reply:
I mean you can use the --label option on the docker build command in the build script and not use label in Dockerfiles.
https://docs.docker.com/engine/reference/commandline/build/

For the moment we have only one Dokerfile but imagine when we will have a lot of Dokerfiles and if we want to change one or add a label.

Reply:
Yes, I understand your concern, i am proposing to modify the `Dockerfile` as follows:

```Dockerfile
# Build-time metadata as defined at http://label-schema.org
ARG BUILD_DATE
ARG VCS_REF
ARG VERSION
ARG VENDOR
ARG SCHEMA_VERSION
ARG VCS_URL
ARG URL
ARG SERVICE
LABEL org.label-schema.build-date=$BUILD_DATE \
      org.label-schema.name="openshift-$SERVICE" \
      org.label-schema.description="OpenShift-compatible $SERVICE Docker image" \
      org.label-schema.url=$URL \
      org.label-schema.vcs-ref=$VCS_REF \
      org.label-schema.vcs-url=$VCS_URL \
      org.label-schema.vendor=$VENDOR \
      org.label-schema.version=$VERSION \
      org.label-schema.schema-version=$SCHEMA_VERSION
```

and add new `--build-arg ARG` to the `docker build` command.

Is this a suitable solution?

Reply:
I think I will add all build arguments in the same `ARG` statement.

Reply:
As discussed with @cchaudier it's way more practical to add labels at the build time (_e.g._ `docker build --label foo`) to avoid polluting the `Dockerfile`.

Reply:
Maybe it depends on which label?
In the Dockerfile : labels that we want to enforce like author, description, etc.
Passed at build time: commit hash, version tag, etc.

Reply:
Indeed. This would avoid having to add ARG statement.